### DISCUSSION TITLE
All available Triggers

### DISCUSSION THREAD

**Discussion by konsalex** [supabase/supabase#2731] [Questions] [answered]
Hey people.

Is there a place where the triggers that we implemented are listed?

Because I seem unable to find for example the trigger I implemented for user sign-up (update the public.user table).

**Comment by cobblestone-fsh** [comment]
I can see mine by clicking "database" on the left and then "triggers" (under alpha preview)

**Reply by konsalex** [reply]
If the trigger is created with code like here: [https://github.com/supabase/supabase/discussions/306](url)

the trigger panel is empty 🤔
![image](https://user-images.githubusercontent.com/12672541/128352842-8284fa25-e339-4c62-83bf-9b7a8724ac2d.png)

**Comment by silentworks** [comment, marked as answer]
This is due to the triggers part of the dashboard not being able to access non-public schemas, this is something the Supabase team are aware of and will be fixing at some point in the future. If you create a trigger based on any table that is available in the `public` schema you will see them inside of the Triggers tab.

But for now, you can go to the SQL tab and use the following to view all triggers.

```sql
SELECT event_object_table,trigger_name,event_manipulation,action_statement,action_timing FROM information_schema.triggers ORDER BY event_object_table,event_manipulation
```
source: https://serverfault.com/a/466732
